Components/Account/Shared/TrainerAppointmentList.razor

```diff
@@ -0,0 +1,164 @@
+﻿@inject ApplicationDbContext DbContext;
```

Comment:
`Pages/TrainerAppointmentList.razor` has changed from prior commits.  Make sure that this moved file has the changes to the original file, notably the `@code{ }` section.